applications/asset_tracker/src/main.c

```diff
@@ -1750,7 +1750,7 @@ static void ui_evt_handler(struct ui_evt evt)
 
 void handle_bsdlib_init_ret(void)
 {
-#if defined(CONFIG_BSD_LIBRARY)
+#if defined(CONFIG_BSD_LIBRARY) && !IS_ENABLED(CONFIG_NRF_CLOUD_FOTA)
```

Comment:
This seems to change something more fundamental with the application but I understand why it is needed.